catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/gmti/GmtiMetacardType.java

```diff
@@ -58,5 +59,6 @@ public void initDescriptors() {
         descriptors.addAll(getDescriptors(CsdidaAttribute.getAttributes()));
         descriptors.addAll(getDescriptors(CsexraAttribute.getAttributes()));
         descriptors.addAll(getDescriptors(HistoaAttribute.getAttributes()));
+        descriptors.addAll(getDescriptors(PiatgbAttribute.getAttributes()));
```

Comment:
Is there a "base" `MetacardType` that you could add these to instead of duplicating across many `MetacardType`s?